### DISCUSSION TITLE
Sketch hangs at mpu.initialize() call on ESP32

### DISCUSSION THREAD

**Discussion by JoeCool66** [jrowberg/i2cdevlib#677] [Q&A]
NM, I found the problem was in the Arduino IDE.

**Comment by valnenov** [comment]
I have the same problem. The https://github.com/ElectronicCats/mpu6050/tree/master/examples/MPU6050_DMP6
sketch hangs up at mpu.initialize() on Adafruit ESP32-S3. 

JoeCool66 What exactly is the problem in the Arduino IDE that you mention and how did you solve it?
